docs/index.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <head>
-<meta http-equiv="refresh" content="0; url=html/index.html" />
+<meta http-equiv="refresh" content="0; url=hmtl/index.html" />
```

Comment:
not sure what happened here but can probably drop the changes to this file from the PR

Reply:
Thanks--I think this is from when I was trying to get the docs working. I'm not sure how to drop the changes though.

Reply:
should I just edit it on my local repo, push it up and do another PR? or is there a better way